sympy/assumptions/assume.py

```diff
@@ -50,27 +80,42 @@ def _sympystr(self, printer):
 
 
 class AppliedPredicate(Boolean):
-    """The class of expressions resulting from applying a Predicate.
+    """
+    The class of expressions resulting from applying ``Predicate`` to
+    the arguments. ``AppliedPredicate`` merely wraps its argument and
+    remain unevaluated. To evaluate it, use ``ask`` function.
 
     Examples
     ========
 
-    >>> from sympy import Q, Symbol
-    >>> x = Symbol('x')
-    >>> Q.integer(x)
-    Q.integer(x)
-    >>> type(Q.integer(x))
+    >>> from sympy import Q, ask
+    >>> Q.integer(1)
+    Q.integer(1)
+
+    ``func`` attribute returns the predicate, and ``args`` attribute
+    returns the argument.
+
+    >>> type(Q.integer(1))
     <class 'sympy.assumptions.assume.AppliedPredicate'>
+    >>> Q.integer(1).func
+    Q.integer
+    >>> Q.integer(1).args
+    (1,)
+
+    Applied predicate can be evaluated to boolean value.
+
+    >>> ask(Q.integer(1))
+    True
 
     """
     __slots__ = ()
 
-    def __new__(cls, predicate, arg):
-        arg = _sympify(arg)
-        return Boolean.__new__(cls, predicate, arg)
-
     is_Atom = True  # do not attempt to decompose this
 
+    def __new__(cls, predicate, *args):
+        args = Tuple(*[_sympify(a) for a in args])
+        return super().__new__(cls, predicate, args)
```

Comment:
Why is this flattening `*args` into a single `Tuple` arg?

Should `predicate` be sympified or otherwise checked in some way?